src/Features/Lsif/GeneratorTest/FoldingRangeTests.vb

```diff
@@ -0,0 +1,115 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.VisualStudio.LanguageServer.Protocol
+Imports Roslyn.Test.Utilities
+Imports Roslyn.Utilities
+
+Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
+    <UseExportProvider>
+    Public NotInheritable Class FoldingRangeTests
+        Private Const TestProjectAssemblyName As String = "TestProject"
+
+        ' Expected 'FoldingRangeKind' argument would likely change for some of the tests when
+        ' https://github.com/dotnet/roslyn/projects/45#card-20049168 is implemented.
```

Comment:
Always appreciate these comments so somebody knows down the road when things change they should change the tests, not the product code.